packages/grafana-data/package.json

```diff
@@ -40,5 +40,6 @@
     "rollup-plugin-visualizer": "0.9.2",
     "sinon": "1.17.6",
     "typescript": "3.5.3"
-  }
+  },
+  "typings": "src/index.ts"
```

Comment:
We can't use typings here. We need to use "types" instead even though those two are synonymous. That's because when we build packages, we also update package.json's `types` field to point to the correct types definition file: https://github.com/grafana/grafana/blob/master/packages/grafana-toolkit/src/cli/tasks/package.build.ts#L45

All should be fine if you change `typings` -> `types`

Reply:
Cool so this will be there really only for dev. Will change that.